django/core/management/commands/listurls.py

```diff
@@ -0,0 +1,134 @@
+import re
+from importlib import import_module
+from io import StringIO
+
+from django.contrib.admindocs.views import (
+    extract_views_from_urlpatterns,
+    simplify_regex,
+)
+from django.core.management.base import BaseCommand
+
+
+def replace_simple_regex(value, offset):
+    if value == "<var>":
+        return "<arg:{}>".format(offset)
+    else:
+        just_value = value.strip("<").strip(">")
+        return "<kwarg:{}>".format(just_value)
+
+
+class Command(BaseCommand):
+    help = """Displays a list of urls used in the project."""
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "url_prefix",
+            nargs="*",
+            help="Only list URLs with these URL prefixes.",
+        )
+
+        parser.add_argument(
```

Comment:
Do we need to have multiple output formats? I couldn't see any discussion on this, so apologies if I've missed something here. 

The table option seems closer to what django-extensions `show_urls` does?

Reply:
The table option is what `show_urls` did, but the long form is what @hippietilley put in their original patch; given I was carrying on their work, it seemed polite to include their format. I'm happy to reverse the defaults, or elide it, but I didn't want to be rude about the work they already did.